tests/test_edgeql_select.py

```diff
@@ -2530,6 +2528,28 @@ async def test_edgeql_select_setops_19(self):
             {'1', '2', '3', '4'},
         )
 
+    async def test_edgeql_select_setops_20(self):
+        res = await self.con.query(r'''
+            SELECT (
+                {(SELECT Issue.time_spent_log.body FILTER false), 'asdf'},
+                Issue,
+            )
+        ''')
+        self.assertEqual(len(res), 4)
+        for row in res:
+            self.assertNotEqual(row[1].id, None)
+
+    async def test_edgeql_select_setops_21(self):
+        res = await self.con.query(r'''
+            SELECT (
+                'oh no' ?? (SELECT Issue.time_spent_log.body FILTER false),
```

Comment:
Why is the order of `??` operands like that? Normally I'd expect to put the "potentially empty set" as the first operand and the "backup" as the second.

Reply:
Yeah, indeed that would make more sense as a useful query, but this is the order that was broken :P

(In a more practical broken query, I guess both sides would be potentially empty)